@navikt/ds-react/grid/src/Cell.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { forwardRef, HTMLAttributes } from "react";
+import cl from "classnames";
+import "@navikt/ds-css/grid/index.css";
+
+type Column = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+export interface CellProps extends HTMLAttributes<HTMLDivElement> {
+  small: Column;
+  medium?: Column;
+  large?: Column;
+  xLarge?: Column;
```

Comment:
Er det noen fordeler med disse navnene over eks de mer konvensjonelle `sm, md, ld, xl`?

Reply:
Hehe, endre akkurat fra sm, md, ld, xl til small, medium, large og xLarge etter forespørsel fra @andnorda 😆

Reply:
lol, ass. To mot én her, altså... La oss gå for `xs, sm, md, lg, xl, xxl`! Endelig svar avgitt. (beklager ekstraarbeid, @mjansrud)